README.md

```diff
@@ -59,12 +59,20 @@ Setup instructions:
 - [Deb packages](https://cloudsmith.io/~gitguardian/repos/ggshield/setup/#formats-deb)
 - [RPM packages](https://cloudsmith.io/~gitguardian/repos/ggshield/setup/#formats-rpm)
 
-## Other Operating Systems - Using pip
+## Other Operating Systems - Using pipx or pip
 
 ### Installing
 
+The recommended way to install `ggshield`` is to use [pipx](https://pypa.github.io/pipx/), which will install it an isolated environment:
+
+```shell
+$ pipx install ggshield
+```
+
+Alternatively, you can install with pip as an user package. This will not work if your Python installation is declared as externally managed (for example if using the system Python on some operating systems):
```

Comment:
```suggestion
Alternatively, you can install with pip as a user package. This will not work if your Python installation is declared as externally managed (for example when using the system Python on operating systems like Debian 12):
```